rita_client/src/exit_manager/mod.rs

```diff
@@ -170,6 +187,18 @@ fn linux_setup_exit_tunnel(
     Ok(())
 }
 
+/// From a exit list, find the corresponding wg public key of our current connected exit to
+/// set up wg tunnels
+fn get_exit_pubkey(ip: IpAddr, exit_list: &ExitList) -> WgKey {
+    for id in exit_list.exit_list.clone() {
+        if id.mesh_ip == ip {
+            return id.wg_public_key;
+        }
+    }
+
+    panic!("Unable to find a valid wg key for current exit, please check that all exit Identities are properly setup on the exit");
```

Comment:
we should consider adding some sort of integrity check for the ExitList struct so that we can error out right away when we get an invalid exit list or at least print a warning.

Reply:
I added a check, should I remove the panic?